mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/ValidateMojo.java

```diff
@@ -59,9 +72,28 @@ protected void validateMavenEnvironment() throws ValidationException {
     }
   }
 
+  protected void validateNotAllowedDependencies() throws ValidationException {
```

Comment:
Is there any special reason why this is not part of the AbstractProjectValidator class?

Reply:
this method in particular is because we transform from project to ArtifactCoordinates in that way the classes in the packager do not deal with maven classes. 

That said the actual validation is done in org.mule.tools.api.validation.AllowedDependencyValidator#areDependenciesAllowed

It's in it's own class for it's easier to test and maintain, I'd keep it there.
But if you feel strong abut it I can  add static method in AbsgractProjectValidator which ends up calling the other class.